ydb workload kv run: add workload type upsert-seq (#2084)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-cli/workload-kv.md

This load test runs several types of load:
* [upsert](#upsert-kv): Using the UPSERT operation, inserts rows that are tuples (key1, key2, ... keyK, value1, value2, ... valueN) into the table created previously with the init command, the K and N numbers are specified in the settings.

> * [upsert-seq](#upsert-seq-kv): variant of upsert load, but inserts rows by sequential keys.

Reviewer: ```suggestion
* [upsert-seq](#upsert-seq-kv): a variant of the upsert load that inserts rows with sequential keys.
```

---|---
`--len` | The size of the rows in bytes that are inserted into the table as values. Default: 8.

## Upsert-seq load {#upsert-seq-kv}

Upsert load variant for inserting rows by sequential keys instead of random.

> Sequential upserts generate a concentrated load on a table (at each point in time) instead of creating a uniform load on the entire table.

Reviewer: ```suggestion
Sequential upserts generate a concentrated load on a small number of table partitions (at each point in time) instead of creating a uniform load on the entire table.
```

> ## Upsert-seq load {#upsert-seq-kv}
> 
> Upsert load variant for inserting rows by sequential keys instead of random.
> 
> Sequential upserts generate a concentrated load on a small number of table partitions (at each point in time) instead of creating a uniform load on the entire table.

Reviewer: ```suggestion
## Upsert-seq load {#upsert-seq-kv}

An upsert load variant that inserts rows using sequential keys instead of random keys.

Sequential upserts concentrate the load on a small number of table partitions at any given time rather than distributing it uniformly across the entire table.
```

To run this type of load, execute the command:

```bash
{{ ydb-cli }} workload kv run upsert-seq [global workload options...] [specific workload options...]
```

> * `global workload options`: [The global options for all types of load](#global-workload-options).
> * `specific workload options`: [Options of a specific load type](#upsert-seq-options).

Reviewer: ```suggestion
* `global workload options`: [The global options for all load types](#global-workload-options).
* `specific workload options`: [Options of a specific load type](#upsert-seq-options).
```

> * `global workload options`: [The global options for all load types](#global-workload-options).
> * `specific workload options`: [Options of a specific load type](#upsert-seq-options).

Reviewer: ```suggestion
* `global workload options`: [global options for all load types](#global-workload-options).
* `specific workload options`: [options for this specific load type](#upsert-seq-options).
```

### Parameters for upsert {#upsert-seq-options}

> Parameter name | Parameter description
> ---|---
> `--len` | The size of the rows in bytes that are inserted into the table as values. Default: 8.
> `--start-first-key` | The starting value for the first column of the primary key. Default: 0.

Reviewer: ```suggestion
Parameter name | Parameter description
---|---
`--len` | The size of the rows in bytes inserted into the table as values. Default: 8.
`--start-first-key` | The starting value for the primary key's first column. Default: 0.
```

> ### Parameters for upsert {#upsert-seq-options}
> 
> Parameter name | Parameter description
> ---|---
> `--len` | The size of the rows in bytes inserted into the table as values. Default: 8.
> `--start-first-key` | The starting value for the primary key's first column. Default: 0.

Reviewer: ```suggestion
### Parameters for upsert-seq {#upsert-seq-options}

Parameter name | Parameter description
---|---
`--len` | The size of the rows, in bytes, inserted into the table as values. Default: 8.
`--start-first-key` | The starting value for the primary key's first column. Default: 0.
```
